pkg/image/utilities/utilities.go

```diff
@@ -141,23 +122,20 @@ func GatherTargetImagesAndSources(chartsPath string, imagesFromArgs []string) (I
 		RancherVersion:  rancherVersion,
 		GithubEndpoints: img.ExtensionEndpoints,
 	}
-	targetImages, targetImagesAndSources, err := img.GetImages(exportConfig, externalLinuxImages, linuxImagesFromArgs, linuxInfo.RKESystemImages)
+	targetImages, targetImagesAndSources, err := img.GetImages(exportConfig, externalLinuxImages, linuxImagesFromArgs)
 	if err != nil {
 		return ImageTargetsAndSources{}, err
 	}
 
 	exportConfig.OsType = img.Windows
-	targetWindowsImages, targetWindowsImagesAndSources, err := img.GetImages(exportConfig, nil, []string{winsAgentUpdateImage}, windowsInfo.RKESystemImages)
```

Comment:
Do the required windows images still get listed in `rancher-windows-images.txt`? Can run locally to compare against images.txt for https://github.com/rancher/rancher/releases/tag/v2.12.0-alpha9 to verify.

Reply:
Discussed offline with @krunalhinguu, the generated windows-images list looks good. `rancher/wins` is the most critical one here, can discuss if other images need to be cleaned up later as it's not specific to RKE1. 

Updated list after removing RKE1 images: 
```
rancher/fleet-agent:v0.13.0-alpha.5
rancher/fluent-bit:3.1.8
rancher/ui-plugin-catalog:4.0.1
rancher/wins:v0.5.1
```
2.12.0-alpha9
```
rancher/fleet-agent:v0.13.0-alpha.5
rancher/fluent-bit:3.1.8
rancher/hyperkube:v1.30.13-rancher1
rancher/hyperkube:v1.31.9-rancher1
rancher/hyperkube:v1.32.5-rancher1
rancher/mirrored-pause:3.7
rancher/rke-tools:v0.1.113
rancher/ui-plugin-catalog:4.0.1
rancher/wins:v0.5.1
```